proc/utils.go

```diff
@@ -56,6 +56,10 @@ type TestSource struct {
 	idx     int
 }
 
+func NewTestSource(batches []zson.Batch) *TestSource {
```

Comment:
This kind of stuff should ideally be in separate PR (or different commit in this PR would work if we didn't squash commits).

Reply:
I was a little lazy here but my `TopProc` pr which is dependent on this, needed this.

Reply:
And I figured better to include this then not have a unit test for the other pr.

Reply:
I'm with @henridf because, as far as I can tell, you could write sort_test.go without doing any surgery on this file.

Reply:
But I won't be cranky if you ignore me!

Reply:
I'm ignoring you guys now just because I'm trying to gsd the first release of zq, but your concerns are duly noted.